src/main/java/edu/cornell/kfs/module/ar/document/service/impl/CuContractsGrantsInvoiceDocumentServiceImpl.java

```diff
@@ -549,5 +552,29 @@ public void recalculateObjectCodeByCategory(ContractsGrantsInvoiceDocument contr
         super.recalculateObjectCodeByCategory(contractsGrantsInvoiceDocument, invoiceDetail, total, invoiceDetailAccountObjectCodes);
     }
     
+    @Override
+    protected String getRecipientAccountNumber(List<InvoiceAccountDetail> accountDetails) {
```

Comment:
I could not find where this method is being used

Reply:
This method is being called on line 93 of this file.

Reply:
Thank you for explaining, that looks good.